app/src/main/res/values/strings.xml

```diff
@@ -284,7 +285,7 @@
     <string name="magic_keyboard_activate_setting_path_2_text">or (\"Settings\" → \"Language &amp; input\" → \"Virtual keyboard\" and pick one.)</string>
     <string name="keyboards_choose_magikeyboard_text">Choose the Magikeyboard when you need to fill a form.</string>
     <string name="keyboards_swicth_magikeyboard_text">You can easily switch from your main keyboard to Magikeyboard with the language button of your keyboard, a long press on the space bar of your keyboard, or, if it is not available, with : </string>
-    <string name="keyboard_select_entry_text">Select your entry with the key.</string>
+    <string name="keyboard_select_entry_text">Select your entry with the keyfile.</string>
```

Comment:
The key is designated by the image just below the text, it is not a key file ...